src/commands/create_release.yml

```diff
@@ -18,8 +18,16 @@ parameters:
     description: The name of this release in Instana.
   release_scope:
     type: string
-    description: >
-      JSON-encoded scoping information for the release.
+    description: |
+      JSON-encoded scoping information for the release. 
+      It can now have environments nests in the string to be replaced like below:
```

Comment:
**suggestion**: The usage of environment variables is supported. Example:

Reply:
@timoinstana could you please redo de review?